<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProcessingClass</class>
 <widget class="QMainWindow" name="ImageProcessingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageProcessing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1088</width>
         <height>731</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="imageLabel">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>image</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Video"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="actionBinary"/>
    <addaction name="actionBlur_image"/>
    <addaction name="actionHSV_Image"/>
    <addaction name="actionSegment_Image"/>
    <addaction name="separator"/>
    <addaction name="actionLaplace_filter"/>
    <addaction name="actionCanny_filter"/>
    <addaction name="separator"/>
    <addaction name="actionEqualize_Histograms"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInfomation"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRotate"/>
    <addaction name="actionFlip"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_out"/>
    <addaction name="separator"/>
    <addaction name="actionHistogram"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProcess"/>
   <addaction name="menuView"/>
   <addaction name="menuImage"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionInfomation">
   <property name="text">
    <string>Infomation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionRotate_180">
   <property name="text">
    <string>Rotate 180Â°</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionBinary">
   <property name="text">
    <string>Binary Image</string>
   </property>
  </action>
  <action name="actionLaplace_filter">
   <property name="text">
    <string>Laplace Filter</string>
   </property>
  </action>
  <action name="actionBlur_image">
   <property name="text">
    <string>Blur Image</string>
   </property>
  </action>
  <action name="actionCanny_filter">
   <property name="text">
    <string>Canny Filter</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="text">
    <string>Histogram</string>
   </property>
  </action>
  <action name="actionOpen_Video">
   <property name="text">
    <string>Open Video</string>
   </property>
  </action>
  <action name="actionFlip">
   <property name="text">
    <string>Flip</string>
   </property>
  </action>
  <action name="actionFlip_Y_axis">
   <property name="text">
    <string>Flip Y-axis</string>
   </property>
  </action>
  <action name="actionEqualize_Histograms">
   <property name="text">
    <string>Equalize Histograms</string>
   </property>
  </action>
  <action name="actionHSV_Image">
   <property name="text">
    <string>HSV Image</string>
   </property>
  </action>
  <action name="actionSegment_Image">
   <property name="text">
    <string>Segment Image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ImageProcessing.qrc"/>
 </resources>
 <connections/>
</ui>
